scripts/apps/desks/directives/StageItemListDirective.js

```diff
@@ -206,12 +206,18 @@ export function StageItemListDirective(search, api, superdesk, desks, cards, $ti
                 }
             };
             scope.fetchPrevious = function() {
-                if (!scope.fetching && scope.page > 2) {
+                if (!scope.fetching && scope.page > 2 && criteria.source.from > 0) {
                     scope.fetching = true;
                     scope.page -= 1;
-                    criteria.source.from = scope.page > 3 ? (scope.page - 3) * criteria.source.size : 0;
                     scope.loading = true;
 
+                    if (scope.page > 3) {
+                        criteria.source.from = (scope.page - 2) * criteria.source.size;
+                    } else {
+                        criteria.source.size *= 2;
```

Comment:
issue is this will only grow, it's never reset back. and because it is used to compute `from` it would skip some items later on. better keep this constant